shared/chat/inbox/row/chat-filter-row/index.js

```diff
@@ -162,22 +167,25 @@ const styleContainer = {
   position: 'relative',
 }
 
-const styleFilterContainer = {
-  ...globalStyles.flexBoxRow,
-  alignItems: 'center',
-  backgroundColor: globalColors.black_05,
-  borderRadius: 19,
-  flexGrow: 1,
-  height: 24,
-  justifyContent: 'center',
-  marginRight: globalMargins.small,
-}
-
-const styleFilterContainerMobile = {
-  ...styleFilterContainer,
-  height: 32,
-  marginRight: globalMargins.small,
-}
+const styleFilterContainer = collapseStyles([
```

Comment:
we should use styleSheetCreate, only merge desktopStyles.editable if isElectron

Reply:
Done.